src/libs/logos.js

```diff
@@ -1,30 +1,35 @@
 import { div, img } from 'react-hyperscript-helpers'
 import anvilLogoWhite from 'src/images/brands/anvil/ANVIL-Logo-White.svg'
 import anvilLogo from 'src/images/brands/anvil/ANVIL-Logo.svg'
+import bioDataCatalystLogo from 'src/images/brands/bioDataCatalyst/bioDataCatalyst-Logo-color.svg'
+import bioDataCatalystLogoWhite from 'src/images/brands/bioDataCatalyst/bioDataCatalyst-Logo-white.svg'
 import datastageLogoWhite from 'src/images/brands/datastage/DataSTAGE-Logo-White.svg'
 import datastageLogo from 'src/images/brands/datastage/DataSTAGE-Logo.svg'
 import fcLogoWhite from 'src/images/brands/firecloud/FireCloud-Logo-White.svg'
 import fcLogo from 'src/images/brands/firecloud/FireCloud-Logo.svg'
 import terraLogoWhite from 'src/images/brands/terra/logo-grey.svg'
 import terraLogoShadow from 'src/images/brands/terra/logo-wShadow.svg'
 import terraLogo from 'src/images/brands/terra/logo.svg'
-import { isAnvil, isDatastage, isFirecloud, isTerra } from 'src/libs/config'
+import { isAnvil, isBioDataCatalyst, isDatastage, isFirecloud, isTerra } from 'src/libs/config'
 import * as Utils from 'src/libs/utils'
 
 
 export const getAppName = (longName = false) => Utils.cond(
   [isFirecloud(), 'FireCloud'],
   [isDatastage(), 'DataStage'],
   [isAnvil(), longName ? 'The NHGRI AnVIL (Genomic Data Science Analysis, Visualization, and Informatics Lab-space)' : 'AnVIL'],
+  [isBioDataCatalyst(), 'NHLBI BioData CATALYST'],
```

Comment:
Should be `'NHLBI BioData Catalyst'`.
Unlike the logo, when the words are written out, "Catalyst" is not all uppercase. (https://www.nhlbidatastage.org/)

Reply:
It's in all capital letters in the inVision docs (can send you a screenshot if you don't have permissions)

Reply:
I checked with Lou who checked their style guide and found that it shouldn't be YELLING. He updated the mocks.